nixos/modules/services/networking/spruned.nix

```diff
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.spruned;
+
+  cliArgs = "--datadir ${cfg.dataDir} --network bitcoin.${cfg.network} ${cfg.extraArguments}";
+in
+{
+  options = {
+    services.spruned = {
+      enable = mkOption {
```

Comment:
`enable = mkEnableOption "the spruned lightweight Bitcoin pseudonode daemon service";`